samples/stock/demo/all-indicators/demo.js

```diff
@@ -0,0 +1,193 @@
+        },
+        legend: {
+            enabled: true
+        },
+        rangeSelector: {
+            selected: 2
+        },
+        tooltip: {
+            split: true
+        },
+        yAxis: [{
+            height: '70%'
+        }, {
+            top: '75%',
+            height: '25%'
+        }],
+        series: [{
+            type: 'candlestick',
+            id: 'aapl',
+            name: 'AAPL',
+            data: data
+        }, {
+            type: 'column',
+            id: 'volume',
+            name: 'Volume',
+            data: volume,
+            yAxis: 1
+        }]
+    });
+
+    document.getElementById("overlays").addEventListener("change", function (e) {
+        const seriesToAdd = e.target.value;
+
+        // Destroy previously existing series.
+        if (chart.series[2] && chart.series[2].yAxis.options.index === 0) {
+            chart.series[2].destroy(false);
+        }
+        if (chart.series[3] && chart.series[3].yAxis.options.index === 0) {
+            chart.series[3].destroy(false);
+        }
+
+        if (seriesToAdd !== 'none') {
+            if (seriesToAdd === 'ikh') {
+                chart.addSeries({
+                    type: 'ikh',
+                    linkedTo: 'aapl',
+                    showInLegend: true,
+                    tenkanLine: {
+                        styles: {
+                            lineColor: 'lightblue'
+                        }
+                    },
+                    kijunLine: {
+                        styles: {
+                            lineColor: 'darkred'
+                        }
+                    },
+                    chikouLine: {
+                        styles: {
+                            lineColor: 'lightgreen'
+                        }
+                    },
+                    senkouSpanA: {
+                        styles: {
+                            lineColor: 'green'
+                        }
+                    },
+                    senkouSpanB: {
+                        styles: {
+                            lineColor: 'red'
+                        }
+                    },
+                    senkouSpan: {
+                        color: 'rgba(0, 255, 0, 0.3)',
+                        styles: {
+                            fill: 'rgba(0, 0, 255, 0.1)'
+                        }
+                    }
+                }, false);
+            } else {
+                chart.addSeries({
+                    type: seriesToAdd,
+                    linkedTo: 'aapl',
+                    showInLegend: true
+                }, false);
+            }
+        }
+        chart.redraw();
+    });
+
+    document.getElementById("oscilators").addEventListener("change", function (e) {
+        const seriesToAdd = e.target.value;
+
+        // Add third axis if needed.
+        if (chart.yAxis.length === 3) {
+            chart.yAxis[0].update({
+                height: '55%'
+            });
```

Comment:
When using multiple updates, set `redraw=false` (the same applies to the `update()` and `addAxis()` below)